src/pd/mod.rs

```diff
@@ -69,112 +69,112 @@ impl Deref for RegionInfo {
 
 pub const INVALID_ID: u64 = 0;
 
-// Client to communicate with placement driver (pd) for special cluster.
-// Because now one pd only supports one cluster, so it is no need to pass
-// cluster id in trait interface every time, so passing the cluster id when
-// creating the PdClient is enough and the PdClient will use this cluster id
-// all the time.
+/// PdClient communicates with placement driver (PD) for special cluster.
```

Comment:
What does "special cluster" mean here? Maybe just remove that part.

```suggestion
/// PdClient communicates with Placement Driver (PD).
```